Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/webdriver_manager/core/file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "/usr/lib64/python3.9/zipfile.py", line 1654, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "/usr/lib64/python3.9/zipfile.py", line 1709, in _extract_member
    shutil.copyfileobj(source, target)
  File "/usr/lib64/python3.9/shutil.py", line 205, in copyfileobj
    buf = fsrc_read(length)
  File "/usr/lib64/python3.9/zipfile.py", line 926, in read
    data = self._read1(n)
  File "/usr/lib64/python3.9/zipfile.py", line 994, in _read1
    data += self._read2(n - len(data))
  File "/usr/lib64/python3.9/zipfile.py", line 1029, in _read2
    raise EOFError
EOFError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_PEUGEOT_SEARCH.py", line 20, in <module>
    service = Service(ChromeDriverManager().install())
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/webdriver_manager/chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/webdriver_manager/core/manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/webdriver_manager/core/driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/webdriver_manager/core/driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/webdriver_manager/core/file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/webdriver_manager/core/file_manager.py", line 67, in __extract_zip
    if e.args[0] not in [26, 13] and e.args[1] not in [
IndexError: tuple index out of range
Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_PEUGEOT_SEARCH.py", line 87, in <module>
    driver.get(url)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 356, in get
    self.execute(Command.GET, {"url": url})
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.554
  (Session info: chrome-headless-shell=126.0.6478.126)
Stacktrace:
#0 0x5639497d5c7a <unknown>
#1 0x5639494b8e2c <unknown>
#2 0x5639494a01a6 <unknown>
#3 0x56394949fd85 <unknown>
#4 0x56394949dcc6 <unknown>
#5 0x56394949e33f <unknown>
#6 0x5639494aec67 <unknown>
#7 0x5639494c51be <unknown>
#8 0x5639494ca8ab <unknown>
#9 0x56394949ea8a <unknown>
#10 0x5639494c503e <unknown>
#11 0x563949547679 <unknown>
#12 0x563949528363 <unknown>
#13 0x5639494f8247 <unknown>
#14 0x5639494f8b9e <unknown>
#15 0x56394979c24b <unknown>
#16 0x5639497a02f1 <unknown>
#17 0x563949787afe <unknown>
#18 0x5639497a0e52 <unknown>
#19 0x56394976c79f <unknown>
#20 0x5639497c5638 <unknown>
#21 0x5639497c5810 <unknown>
#22 0x5639497d4dac <unknown>
#23 0x7f912bc9f7f2 start_thread

Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_PEUGEOT_SEARCH.py", line 87, in <module>
    driver.get(url)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 356, in get
    self.execute(Command.GET, {"url": url})
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 296.980
  (Session info: chrome-headless-shell=126.0.6478.126)
Stacktrace:
#0 0x55635408bc7a <unknown>
#1 0x556353d6ee2c <unknown>
#2 0x556353d561a6 <unknown>
#3 0x556353d55d85 <unknown>
#4 0x556353d53cc6 <unknown>
#5 0x556353d5433f <unknown>
#6 0x556353d64c67 <unknown>
#7 0x556353d7b1be <unknown>
#8 0x556353d808ab <unknown>
#9 0x556353d54a8a <unknown>
#10 0x556353d7b03e <unknown>
#11 0x556353dfd679 <unknown>
#12 0x556353dde363 <unknown>
#13 0x556353dae247 <unknown>
#14 0x556353daeb9e <unknown>
#15 0x55635405224b <unknown>
#16 0x5563540562f1 <unknown>
#17 0x55635403dafe <unknown>
#18 0x556354056e52 <unknown>
#19 0x55635402279f <unknown>
#20 0x55635407b638 <unknown>
#21 0x55635407b810 <unknown>
#22 0x55635408adac <unknown>
#23 0x7f4ad949f7f2 start_thread

Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x56425cc44c7a <unknown>
#1 0x56425c927e2c <unknown>
#2 0x56425c974661 <unknown>
#3 0x56425c974751 <unknown>
#4 0x56425c9b8f64 <unknown>
#5 0x56425c9975ed <unknown>
#6 0x56425c9b6303 <unknown>
#7 0x56425c997363 <unknown>
#8 0x56425c967247 <unknown>
#9 0x56425c967b9e <unknown>
#10 0x56425cc0b24b <unknown>
#11 0x56425cc0f2f1 <unknown>
#12 0x56425cbf6afe <unknown>
#13 0x56425cc0fe52 <unknown>
#14 0x56425cbdb79f <unknown>
#15 0x56425cc34638 <unknown>
#16 0x56425cc34810 <unknown>
#17 0x56425cc43dac <unknown>
#18 0x7ff02dc9f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x564d73572c7a <unknown>
#1 0x564d73255e2c <unknown>
#2 0x564d732a2661 <unknown>
#3 0x564d732a2751 <unknown>
#4 0x564d732e6f64 <unknown>
#5 0x564d732c55ed <unknown>
#6 0x564d732e4303 <unknown>
#7 0x564d732c5363 <unknown>
#8 0x564d73295247 <unknown>
#9 0x564d73295b9e <unknown>
#10 0x564d7353924b <unknown>
#11 0x564d7353d2f1 <unknown>
#12 0x564d73524afe <unknown>
#13 0x564d7353de52 <unknown>
#14 0x564d7350979f <unknown>
#15 0x564d73562638 <unknown>
#16 0x564d73562810 <unknown>
#17 0x564d73571dac <unknown>
#18 0x7f82f809f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x5644a70b6c7a <unknown>
#1 0x5644a6d99e2c <unknown>
#2 0x5644a6de6661 <unknown>
#3 0x5644a6de6751 <unknown>
#4 0x5644a6e2af64 <unknown>
#5 0x5644a6e095ed <unknown>
#6 0x5644a6e28303 <unknown>
#7 0x5644a6e09363 <unknown>
#8 0x5644a6dd9247 <unknown>
#9 0x5644a6dd9b9e <unknown>
#10 0x5644a707d24b <unknown>
#11 0x5644a70812f1 <unknown>
#12 0x5644a7068afe <unknown>
#13 0x5644a7081e52 <unknown>
#14 0x5644a704d79f <unknown>
#15 0x5644a70a6638 <unknown>
#16 0x5644a70a6810 <unknown>
#17 0x5644a70b5dac <unknown>
#18 0x7f715de9f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_PEUGEOT_SEARCH.py", line 87, in <module>
    driver.get(url)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 356, in get
    self.execute(Command.GET, {"url": url})
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 288.823
  (Session info: chrome-headless-shell=126.0.6478.126)
Stacktrace:
#0 0x55e223ef3c7a <unknown>
#1 0x55e223bd6e2c <unknown>
#2 0x55e223bbe1a6 <unknown>
#3 0x55e223bbdd85 <unknown>
#4 0x55e223bbbcc6 <unknown>
#5 0x55e223bbc33f <unknown>
#6 0x55e223bccc67 <unknown>
#7 0x55e223be31be <unknown>
#8 0x55e223be88ab <unknown>
#9 0x55e223bbca8a <unknown>
#10 0x55e223be2d76 <unknown>
#11 0x55e223c65679 <unknown>
#12 0x55e223c46363 <unknown>
#13 0x55e223c16247 <unknown>
#14 0x55e223c16b9e <unknown>
#15 0x55e223eba24b <unknown>
#16 0x55e223ebe2f1 <unknown>
#17 0x55e223ea5afe <unknown>
#18 0x55e223ebee52 <unknown>
#19 0x55e223e8a79f <unknown>
#20 0x55e223ee3638 <unknown>
#21 0x55e223ee3810 <unknown>
#22 0x55e223ef2dac <unknown>
#23 0x7f040069f7f2 start_thread

Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_PEUGEOT_SEARCH.py", line 21, in <module>
    driver = webdriver.Chrome(service=service, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__
    super().__init__(
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py", line 66, in __init__
    super().__init__(command_executor=executor, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 208, in __init__
    self.start_session(capabilities)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 292, in start_session
    response = self.execute(Command.NEW_SESSION, caps)["value"]
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: session not created
from chrome not reachable
Stacktrace:
#0 0x557b0dbc6c7a <unknown>
#1 0x557b0d8a9c71 <unknown>
#2 0x557b0d89433f <unknown>
#3 0x557b0d8e4895 <unknown>
#4 0x557b0d8df0a1 <unknown>
#5 0x557b0d8db17b <unknown>
#6 0x557b0d925c49 <unknown>
#7 0x557b0d919363 <unknown>
#8 0x557b0d8e9247 <unknown>
#9 0x557b0d8e9b9e <unknown>
#10 0x557b0db8d24b <unknown>
#11 0x557b0db912f1 <unknown>
#12 0x557b0db78afe <unknown>
#13 0x557b0db91e52 <unknown>
#14 0x557b0db5d79f <unknown>
#15 0x557b0dbb6638 <unknown>
#16 0x557b0dbb6810 <unknown>
#17 0x557b0dbc5dac <unknown>
#18 0x7f19ade9f7f2 start_thread

Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_PEUGEOT_SEARCH.py", line 21, in <module>
    driver = webdriver.Chrome(service=service, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__
    super().__init__(
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py", line 66, in __init__
    super().__init__(command_executor=executor, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 208, in __init__
    self.start_session(capabilities)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 292, in start_session
    response = self.execute(Command.NEW_SESSION, caps)["value"]
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: session not created: DevToolsActivePort file doesn't exist
Stacktrace:
#0 0x5575be0fcc7a <unknown>
#1 0x5575bdddfe2c <unknown>
#2 0x5575bde1a175 <unknown>
#3 0x5575bde152d1 <unknown>
#4 0x5575bde1117b <unknown>
#5 0x5575bde5bc49 <unknown>
#6 0x5575bde4f363 <unknown>
#7 0x5575bde1f247 <unknown>
#8 0x5575bde1fb9e <unknown>
#9 0x5575be0c324b <unknown>
#10 0x5575be0c72f1 <unknown>
#11 0x5575be0aeafe <unknown>
#12 0x5575be0c7e52 <unknown>
#13 0x5575be09379f <unknown>
#14 0x5575be0ec638 <unknown>
#15 0x5575be0ec810 <unknown>
#16 0x5575be0fbdac <unknown>
#17 0x7fc32ec9f7f2 start_thread

Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x561a90e99c7a <unknown>
#1 0x561a90b7ce2c <unknown>
#2 0x561a90bc9661 <unknown>
#3 0x561a90bc9751 <unknown>
#4 0x561a90c0df64 <unknown>
#5 0x561a90bec5ed <unknown>
#6 0x561a90c0b303 <unknown>
#7 0x561a90bec363 <unknown>
#8 0x561a90bbc247 <unknown>
#9 0x561a90bbcb9e <unknown>
#10 0x561a90e6024b <unknown>
#11 0x561a90e642f1 <unknown>
#12 0x561a90e4bafe <unknown>
#13 0x561a90e64e52 <unknown>
#14 0x561a90e3079f <unknown>
#15 0x561a90e89638 <unknown>
#16 0x561a90e89810 <unknown>
#17 0x561a90e98dac <unknown>
#18 0x7efd75e9f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x55f9e6db4c7a <unknown>
#1 0x55f9e6a97e2c <unknown>
#2 0x55f9e6ae4661 <unknown>
#3 0x55f9e6ae4751 <unknown>
#4 0x55f9e6b28f64 <unknown>
#5 0x55f9e6b075ed <unknown>
#6 0x55f9e6b26303 <unknown>
#7 0x55f9e6b07363 <unknown>
#8 0x55f9e6ad7247 <unknown>
#9 0x55f9e6ad7b9e <unknown>
#10 0x55f9e6d7b24b <unknown>
#11 0x55f9e6d7f2f1 <unknown>
#12 0x55f9e6d66afe <unknown>
#13 0x55f9e6d7fe52 <unknown>
#14 0x55f9e6d4b79f <unknown>
#15 0x55f9e6da4638 <unknown>
#16 0x55f9e6da4810 <unknown>
#17 0x55f9e6db3dac <unknown>
#18 0x7f8b47a9f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x562c1d378c7a <unknown>
#1 0x562c1d05be2c <unknown>
#2 0x562c1d0a8661 <unknown>
#3 0x562c1d0a8751 <unknown>
#4 0x562c1d0ecf64 <unknown>
#5 0x562c1d0cb5ed <unknown>
#6 0x562c1d0ea303 <unknown>
#7 0x562c1d0cb363 <unknown>
#8 0x562c1d09b247 <unknown>
#9 0x562c1d09bb9e <unknown>
#10 0x562c1d33f24b <unknown>
#11 0x562c1d3432f1 <unknown>
#12 0x562c1d32aafe <unknown>
#13 0x562c1d343e52 <unknown>
#14 0x562c1d30f79f <unknown>
#15 0x562c1d368638 <unknown>
#16 0x562c1d368810 <unknown>
#17 0x562c1d377dac <unknown>
#18 0x7fa53609f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x562e55c4cc7a <unknown>
#1 0x562e5592fe2c <unknown>
#2 0x562e5597c661 <unknown>
#3 0x562e5597c751 <unknown>
#4 0x562e559c0f64 <unknown>
#5 0x562e5599f5ed <unknown>
#6 0x562e559be303 <unknown>
#7 0x562e5599f363 <unknown>
#8 0x562e5596f247 <unknown>
#9 0x562e5596fb9e <unknown>
#10 0x562e55c1324b <unknown>
#11 0x562e55c172f1 <unknown>
#12 0x562e55bfeafe <unknown>
#13 0x562e55c17e52 <unknown>
#14 0x562e55be379f <unknown>
#15 0x562e55c3c638 <unknown>
#16 0x562e55c3c810 <unknown>
#17 0x562e55c4bdac <unknown>
#18 0x7ff2e029f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x564d763d2c7a <unknown>
#1 0x564d760b5e2c <unknown>
#2 0x564d76102661 <unknown>
#3 0x564d76102751 <unknown>
#4 0x564d76146f64 <unknown>
#5 0x564d761255ed <unknown>
#6 0x564d76144303 <unknown>
#7 0x564d76125363 <unknown>
#8 0x564d760f5247 <unknown>
#9 0x564d760f5b9e <unknown>
#10 0x564d7639924b <unknown>
#11 0x564d7639d2f1 <unknown>
#12 0x564d76384afe <unknown>
#13 0x564d7639de52 <unknown>
#14 0x564d7636979f <unknown>
#15 0x564d763c2638 <unknown>
#16 0x564d763c2810 <unknown>
#17 0x564d763d1dac <unknown>
#18 0x7f06ed69f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x561108ec9c7a <unknown>
#1 0x561108bace2c <unknown>
#2 0x561108bf9661 <unknown>
#3 0x561108bf9751 <unknown>
#4 0x561108c3df64 <unknown>
#5 0x561108c1c5ed <unknown>
#6 0x561108c3b303 <unknown>
#7 0x561108c1c363 <unknown>
#8 0x561108bec247 <unknown>
#9 0x561108becb9e <unknown>
#10 0x561108e9024b <unknown>
#11 0x561108e942f1 <unknown>
#12 0x561108e7bafe <unknown>
#13 0x561108e94e52 <unknown>
#14 0x561108e6079f <unknown>
#15 0x561108eb9638 <unknown>
#16 0x561108eb9810 <unknown>
#17 0x561108ec8dac <unknown>
#18 0x7fa53a89f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x56036fce3c7a <unknown>
#1 0x56036f9c6e2c <unknown>
#2 0x56036fa13661 <unknown>
#3 0x56036fa13751 <unknown>
#4 0x56036fa57f64 <unknown>
#5 0x56036fa365ed <unknown>
#6 0x56036fa55303 <unknown>
#7 0x56036fa36363 <unknown>
#8 0x56036fa06247 <unknown>
#9 0x56036fa06b9e <unknown>
#10 0x56036fcaa24b <unknown>
#11 0x56036fcae2f1 <unknown>
#12 0x56036fc95afe <unknown>
#13 0x56036fcaee52 <unknown>
#14 0x56036fc7a79f <unknown>
#15 0x56036fcd3638 <unknown>
#16 0x56036fcd3810 <unknown>
#17 0x56036fce2dac <unknown>
#18 0x7fdcb689f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x56343c43bc7a <unknown>
#1 0x56343c11ee2c <unknown>
#2 0x56343c16b661 <unknown>
#3 0x56343c16b751 <unknown>
#4 0x56343c1aff64 <unknown>
#5 0x56343c18e5ed <unknown>
#6 0x56343c1ad303 <unknown>
#7 0x56343c18e363 <unknown>
#8 0x56343c15e247 <unknown>
#9 0x56343c15eb9e <unknown>
#10 0x56343c40224b <unknown>
#11 0x56343c4062f1 <unknown>
#12 0x56343c3edafe <unknown>
#13 0x56343c406e52 <unknown>
#14 0x56343c3d279f <unknown>
#15 0x56343c42b638 <unknown>
#16 0x56343c42b810 <unknown>
#17 0x56343c43adac <unknown>
#18 0x7fc1ba69f7f2 start_thread

Cookie consent has been accepted.
Data from 3 pages saved to 'PEUGEOT_SEARCH.json'
